test/api/qcFlags.test.js

```diff
@@ -393,7 +394,7 @@ module.exports = () => {
 
     describe('DELETE /api/qcFlags/:id', () => {
         it('should fail to delete QC flag when being neither owner nor admin', async () => {
```

Comment:
You may add a test for trying to delete verified flag?

Reply:
added